Documentation/ContentElements/EditingContent/Index.rst

```diff
@@ -26,3 +26,9 @@ display different input fields.
 
 This action does not delete content. It is saved in the background and will be
 restored if you select the applicable content element type again.
+
+.. tip::
+
+   If you are more comfortable using shortcuts, there are different kinds of
+   :ref:`keyboard commands <t3editors:keyboard_commands>` you can use in
+   the Backend Editor.
```

Comment:
Seeing this happens multiple times,
how do you think about outsourcing this tip as a rst include file and doing an include of that at the current places?
(I can also do that in a follow-up, but if you'd be interested to do it, that would be a great help!)

Reply:
I could do that, yeah. Is there any convention for where to place that include file? Like in "Documentation/ContentElements/Includes" or just place it in the existing ContentElements Folder next to the Index.rst?

Reply:
In this repo we don't have includes yet. In the Core API docs we often create subdirectories only when more than one file is included.

In your case here I would put a file like "TipHotkeys.rst" or so just into the same directory as Index.rst, yes. We can move it later when more includes happen.

Thanks! :)